docs/en/guides/apply_catboost_model.md

```diff
@@ -0,0 +1,232 @@
+# Applying a Catboost Model in ClickHouse {#applying-catboost-model-in-clickhouse}
+
+[CatBoost](https://catboost.ai) — is a free and open-source gradient boosting library developed at [Yandex](https://yandex.com/company/) for machine learning.
+
+With this instruction, you will learn to apply pre-trained models in ClickHouse: as a result, you run the model inference from SQL.
+
+To apply a CatBoost model in ClickHouse:
+
+1. [Create a Table](#create-table).
+2. [Insert the Data to the Table](#insert-the-data-to-the-table).
+3. [Configure the Model](#configure-the-model).
+4. [Run the Model Inference from SQL](#run-the-model-inference).
+
+For more information about training CatBoost models, see [Training and applying models](https://catboost.ai/docs/features/training.html#training).
+
+## Prerequisites {#prerequisites}
+
+If you don't have the [Docker](https://docs.docker.com/install/) yet, install it.
+
+!!! note "Note"
+    [Docker](https://www.docker.com) is a software platform that allows you to create containers that isolate a CatBoost and ClickHouse installation from the rest of the system.
+
+Before applying a CatBoost model:
+
+**1.** Pull the [Docker image](https://hub.docker.com/r/yandex/tutorial-catboost-clickhouse) from the registry:
+
+```bash
+$ docker pull yandex/tutorial-catboost-clickhouse
+```
+
+This Docker image contains everything you need to run CatBoost and ClickHouse: code, runtime, libraries, environment variables, and configuration files.
+
+**2.** Make sure the Docker image has been successfully pulled:
+
+```bash
+$ docker image ls
+REPOSITORY                            TAG                 IMAGE ID            CREATED             SIZE
+yandex/tutorial-catboost-clickhouse   latest              3e5ad9fae997        19 months ago       1.58GB
+```
+
+**3.** Start a Docker container based on this image:
+
+```bash
+$ docker run -it -p 8888:8888 yandex/tutorial-catboost-clickhouse
+```
+
+**4.** Install packages:
+
+```bash
+$ sudo apt-get install dirmngr
+$ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E0C56BD4
+
+$ echo "deb http://repo.yandex.ru/clickhouse/deb/stable/ main/" | sudo tee /etc/apt/sources.list.d/clickhouse.list
+$ sudo apt-get update
+
+$ sudo apt-get install -y clickhouse-server clickhouse-client
+
+$ sudo service clickhouse-server start
+$ clickhouse-client
+```
+
+For more information, see [Quick Start](https://clickhouse.yandex/#quick-start).
```

Comment:
there's not much additional information in quickstart, more verbose similar guide is over here https://clickhouse.yandex/docs/en/getting_started/